uploader/s3.go

```diff
@@ -153,15 +153,24 @@ func (u *S3Backend) uploadFile(ctx context.Context, local io.Reader, remote stri
 		contentType = defaultS3ContentType
 	}
 
+	contentEncoding := ""
+	if contentType == defaultS3ContentType {
```

Comment:
Shouldn't this just be == "text/plain"?

Reply:
Yea I wasn't sure if I should reuse that constant or not.
But yea I do think that uft-8 does only make sense with text/plain, regardless of what the default content type is in the file.